discord/ext/commands/errors.py

```diff
@@ -971,4 +971,4 @@ class MissingFlagArgument(FlagError):
     """
     def __init__(self, flag):
         self.flag = flag
-        super().__init__(f'Flag {flag.name!r} does not have an argument')
+        super().__init__(f'Flag {flag.name!r} does not have an argument')
```

Comment:
Why did you remove the newline at the end of this file?

Reply:
Because I'm bad.